### PR TITLE
EN-158: Update to new version of datacoordinator.

### PR DESCRIPTION
**Pull request by aerust**
Thinking about this more, I think that the spandex secondary jar needs to be updated and rebuilt. The rebuild of the jar needs to go with a rebuild of secondary watcher. 

edit: Actually needs to go before secondary watcher.

### CODE REVIEW

**Comment by sokrahta** [comment]
rebased to what's on master today, have you got a demo to show?

**Comment by aerust** [comment]
Okay, decided to remove functionality for dropping computation strategies... Though, I now realize that I'm going to make another binary incompatible change to data-coordinator that is going into the same long-lived branch other there.... so at some point here I will probably squash all the commits in this branch and then actually demo it for you once I am ready.

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-version in ThisBuild := "1.1.1-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "2.0.0-SNAPSHOT"
```
Comment (sokrahta) [inline version.sbt]: let's avoid changing the version inside a pr, we can simply use sbt-release to choose the next version numbers

File: spandex-secondary/src/test/scala/com/socrata/spandex/secondary/SpandexSecondaryLikeSpec.scala [hunk 2]
```diff
@@ -105,7 +106,7 @@ class SpandexSecondaryLikeSpec extends FunSuiteLike with Matchers with TestESDat
         new ColumnId(4) -> new SoQLNumber(new BigDecimal(3)))
     )
 
-    secondary.resync(datasetInfo, copyInfo, schema, None, unmanaged(rows.iterator), Seq.empty)
+    secondary.resync(datasetInfo, copyInfo, schema, None, unmanaged(rows.iterator), Seq.empty, isLatestCopy = true)
```
Comment (sokrahta) [inline spandex-secondary/src/test/scala/com/socrata/spandex/secondary/SpandexSecondaryLikeSpec.scala]: are we likely to receive resyncs with `isLatestCopy=false` ?

**Comment by sokrahta** [comment]
thanks for the demo
:sheep: :it:

**Pull request merged**